elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base),
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
+  BorderRadius: pfe-var(ui--border-radius),
+  Color: pfe-var(text),
+  FontSize: pfe-var(font-size),
+  Height: 36px,
+  Padding: 8px 6px,
+);
+
+:host {
+  display: inline-block;
+  position: relative;
+}
+
+::slotted(input) {
+  all: unset;
+  opacity: 0;
+  padding: pfe-local(Padding);
+  border-radius: pfe-local(BorderRadius);
+  border: pfe-local(BorderWidth) solid transparent;
```

Comment:
```suggestion
  border: pfe-local(BorderWidth) pfe-var(ui--border-style) transparent;
```

Reply:
Why style the slotted input?  Isn't the shadow input the only visible one?

Reply:
We need to have the light DOM input in the slot so the browser validation works.

Reply:
Right it needs to be there but isn't it hidden? Can the user see it?

Reply:
The opacity is set to zero so it's there, it's not in the tab index but still focusable (which transfers focus to the shadow dom), and not visually visible.